### PR TITLE
added instructions to change radio channel and address

### PR DESCRIPTION
**Pull request by barzanisar**
@hamishwillee 

Bitcraze crazyflie-lib-python has accepted my PR for the cfbridge.py so I have changed the links for cfbridge to their folder. 

Also, I have added instructions to launch cfbridge with a different radio channel and crazyflie address than their default values.

### CODE REVIEW

File: en/flight_controller/crazyflie2.md [hunk 1]
```diff
@@ -105,24 +112,18 @@ with the crazyradio PA.
...
- * Activate the virtual environment: `source venv-cflib/bin/activate`
-
 To connect crazyflie with crazyradio, **launch cfbridge** by following these steps:
 - Switch on CF2 (which is already flashed with PX4 firmware) by pressing its ON button and wait for it to boot up.
 - Connect a Crazyradio PA via USB.
 - Navigate to the crazyflie-lib-python folder.
 - Activate the environment: `source venv-cflib/bin/activate` or `source venv/bin/activate` depending on the name of the venv folder you have.
 - `cd examples`
 - Launch cfbridge: `python cfbridge.py`
+> **Note** *Cfbridge* by default tries to initiate the radio link communication on channel 80 and with crazyflie address 0xE7E7E7E7E7. If you are using [multiple crazyflies and/or crazyradios](https://github.com/dennisss/cfbridge/blob/master/README.md#advanced-swarming) in the same room and want to use a different channel and/or address for each, first connect the crazyflie with QGC via a USB cable and change the syslink parameters (channel, address) in QGC. Next, launch the cfbridge by giving the same channel and address as the first and second arguments respectively, e.g: `python cfbridge.py 90 0x0202020202`
```
Comment (hamishwillee) [inline en/flight_controller/crazyflie2.md]: I assume this bullet is part of the previous bullet point. Can you please indent 2 spaces?

File: en/flight_controller/crazyflie2.md [hunk 2]
```diff
@@ -105,24 +112,18 @@ with the crazyradio PA.
...
 - Open QGC.
 - After using *cfbridge*, you can deactivate the virtualenv if you activated it by pressing `CTRL+z`. Most of the time, launching *cfbridge* again from the same terminal doesn't connect to crazyflie, this can be solved by closing the terminal and relaunching *cfbridge* in a new terminal. 
 
-> **Tip** If you change any driver in [crazyflie-lib-python](https://github.com/barzanisar/crazyflie-lib-python/tree/cfbridge) 
+> **Tip** If you change any driver in [crazyflie-lib-python](https://github.com/bitcraze/crazyflie-lib-python)
```
Comment (hamishwillee) [inline en/flight_controller/crazyflie2.md]: Is this bullet part of the bullet point? If not, fine. If so, should be indented.

**Comment by hamishwillee** [comment]
Thanks! Minor points inline then this can be merged.

File: en/flight_controller/crazyflie2.md (hunk 2 above)
Comment (barzanisar) [reply en/flight_controller/crazyflie2.md]: Nope, it is a separate point.

**Comment by hamishwillee** [comment]
THanks @barzanisar . Merging now!

**Pull request merged**